core/perf_test/CMakeLists.txt

```diff
@@ -15,102 +15,97 @@ IF ((KOKKOS_ENABLE_OPENMPTARGET OR KOKKOS_ENABLE_OPENACC) AND KOKKOS_CXX_COMPILE
   RETURN()
 ENDIF()
 
+IF(KOKKOS_ENABLE_TESTS)
```

Comment:
We should probably comment about the intent here.  Essentially all the `PerformanceTest_*` executables are considered as being part of the regular tests, not part of the benchmarks.

Reply:
I will add the explanation. Also this should all eventually get ported into benchmarks, which will make this `IF` obsolete.

Reply:
Comment added: 327aac5.